qa/automation/CareerPortal.Tests.Scenarios/samsonova_sendCV/1.1.send_cv.feature

```diff
@@ -0,0 +1,40 @@
+  Feature: 1.1.Send CV
+
+  Scenario: 1.1_01 positive sending CV
+  Given user opened the form "Send CV"
+  When user enters valid data in the field "Name"
+  And user enters valid data in the field "Email"
+  And user attaches CV
+  And user marks "I agree to the personal data processing"
+  And user clicks the "Send" button
+  Then my CV has been send and success message should appear
+
+
+  Scenario: 1.1_02 exceeding the limit of characters in the field "Name"
+  Given user opened the form "Send CV"
+  When user enters 71 characters with space delimiter in the field "Name"
+  Then a message should appear "The maximum number of characters is 70"
+
+  Scenario: 1.1_03 entering the number format data in the field "Name"
+  Given user opened the form "Send CV"
+  When user enters the number format data in the field "Name"
+  Then a message should appear "Invalid name format"
+
+
+  Scenario: 1.1_04 entering invalid data in the field "Email"
+  Given user opened the form "Send CV"
+  When user enters email-address without "@"-character in the field "Email"
+  Then a message should appear "This field is required"
+
+
+  Scenario: 1.1_05 upload a file larger than 10 Mb
```

Comment:
Maybe it will be better in order to testing coverage to add one more test - for example, positive one: attach a CV file in valid size (10 Mb or less)

Reply:
done, i added this in 1.1_01